db/tables.yml

```diff
@@ -87,11 +87,12 @@ denylisted_notification_entities:
   value: jsonb not null
   version: integer not null
   etag: uuid
-cache_purges_entities:
-  partition_key: text not null
-  row_key: text not null
-  value: jsonb not null
-  version: integer not null
+cache_purges:
+  provisioner_id: text not null
+  worker_type: text not null
+  cache_name: text not null
```

Comment:
TIL `text` is not worse performance than `varchar` in postgres. So that's not a reason to use `varchar` here, but I want to make sure that we're actively deciding to limit `worker_type` length at the api schema level rather than the db schema level here? That seems like a reasonable trade-off to make -- is that written down in our best-practices, etc? Just want to know for when I start doing work like this.

Reply:
I think things like the length of `worker_type` could possibly change in the future. Any changes to the db in the future (i.e, altering the column to accept something different) will hang the db until all of the changes have been done. `alter` afaict is not a very fast operation so for large tables this would possibly need a downtime. Agreed, we should include that in the best practices if the team agrees with this.

Reply:
Ok, seems like you+dustin are in agreement about what to do here and so I'm convinced too! Adding it to best-practices at some point makes sense but obviously not blocking landing this.